### PR TITLE
Gradle tasks described in docker documentation

### PR DESCRIPTION
**Pull request by hendrikebbers**

### CODE REVIEW

File: docs/docker-quickstart.md [hunk 1]
```diff
@@ -111,9 +88,9 @@ As you run operations against the local network, each node will periodically sav
 a combination of PostgreSQL tables under _compose-network/pgdata/_ and state files under, for example,
 _compose-network/node0/saved/com.hedera.node.app.service.mono.ServicesMain/0/hedera/_.
 
-To stop the network, use `Ctrl+C` (or `docker-compose stop` if running with detached containers).
+To stop the network, call the `stopDockerContainers` gradle tasks (execute `./gradlew stopDockerContainers` from the root folder of the repo).
```
Comment (mhess-swl) [inline docs/docker-quickstart.md:91]:
```suggestion
To stop the network, call the `stopDockerContainers` gradle task (execute `./gradlew stopDockerContainers` from the root folder of the repo).
```

File: docs/docker-quickstart.md [hunk 2]
```diff
@@ -1,20 +1,14 @@
 # Docker quickstart
 
+To execute the integration and end-to-end tests of this repo a local hedera node need to be started.
```
Comment (mhess-swl) [inline docs/docker-quickstart.md:3]: Nit:
```suggestion
To execute the integration and end-to-end tests of this repo, a local hedera node needs to be started.
```

File: docs/docker-quickstart.md [hunk 3]
```diff
@@ -39,34 +33,23 @@ Second, choose a tag for your build. The tag will be added
...
+`"oa-release-r5-rc6-13-gf18d2ff77-dirty"`. 
+Ensure the Docker Compose .env file `(../docker/.env)` has an empty registry prefix and your tag:
 
 ```
 TAG=oa-release-r5-rc6-13-gf18d2ff77-dirty
 REGISTRY_PREFIX=
 ```
 
-Third, build the image with an empty registry prefix and the `TAG` from your `.env` file:
+The file can be created by calling the `updateDockerEnv` gradle tasks (execute `./gradlew updateDockerEnv` from the root folder of the repo).
```
Comment (mhess-swl) [inline docs/docker-quickstart.md:44]:
```suggestion
The file can be created by calling the `updateDockerEnv` gradle task (execute `./gradlew updateDockerEnv` from the root folder of the repo).
```

File: docs/docker-quickstart.md [hunk 4]
```diff
@@ -39,34 +33,23 @@ Second, choose a tag for your build. The tag will be added
...
 
-```
-docker build -t services-node:oa-release-r5-rc6-13-gf18d2ff77-dirty .
-```
+Third, build the image with an empty registry prefix and the `TAG` from your `.env` file by calling the `createDockerImage` gradle tasks (execute `./gradlew createDockerImage` from the root folder of the repo).
```
Comment (mhess-swl) [inline docs/docker-quickstart.md:46]:
```suggestion
Third, build the image with an empty registry prefix and the `TAG` from your `.env` file by calling the `createDockerImage` gradle task (execute `./gradlew createDockerImage` from the root folder of the repo).
```

File: docs/docker-quickstart.md [hunk 5]
```diff
@@ -39,34 +33,23 @@ Second, choose a tag for your build. The tag will be added
...
+This is a multi-stage build that could take **several minutes**, depending on your environment.
 
 ## Starting the Compose network
 
-Run:
-
-```
-docker-compose up
-```
+The network can be started by calling the `startDockerContainers` gradle tasks (execute `./gradlew startDockerContainers` from the root folder of the repo).
```
Comment (mhess-swl) [inline docs/docker-quickstart.md:52]:
```suggestion
The network can be started by calling the `startDockerContainers` gradle task (execute `./gradlew startDockerContainers` from the root folder of the repo).
```

**Review by mhess-swl — APPROVED** [review]

File: docs/docker-quickstart.md [hunk 6]
```diff
@@ -39,34 +33,23 @@ Second, choose a tag for your build. The tag will be added
...
+`"oa-release-r5-rc6-13-gf18d2ff77-dirty"`. 
+Ensure the Docker Compose .env file `(../docker/.env)` has an empty registry prefix and your tag:
 
 ```
 TAG=oa-release-r5-rc6-13-gf18d2ff77-dirty
 REGISTRY_PREFIX=
 ```
 
-Third, build the image with an empty registry prefix and the `TAG` from your `.env` file:
+The file can be created by calling the `updateDockerEnv` gradle task (execute `./gradlew updateDockerEnv` from the root folder of the repo).
```
Comment (netopyr) [inline docs/docker-quickstart.md:44]: I prefer to have commands for the terminal in a separate line (within a code block).

File: docs/docker-quickstart.md [hunk 7]
```diff
@@ -93,13 +76,7 @@ Public: 0aa8e21064c61eab86e2a9c164565b4e7a9a4146106e0a6cd03a8c395a110e92
...
 ```
 
-You can now run operations against your local network using any HAPI client. For example:
-
-```
-./mvnw install -DskipTests
-cd test-clients
-../mvnw exec:java -Dexec.mainClass=com.hedera.services.bdd.suites.compose.LocalNetworkCheck -Dexec.cleanupDaemonThreads=false
-```
+You can now run the integration and end-to-end tests. In general you can run any operation against your local network by using any HAPI client.
```
Comment (netopyr) [inline docs/docker-quickstart.md:79]: Is it possible to provide a simple example with Gradle?

**Review by netopyr — APPROVED** [review]
Added some minor comments that can be handled later.

**Pull request closed without merging**